packs/jenkins/vars_cloud.nomad

```diff
@@ -0,0 +1,79 @@
+docker_jenkins_env_vars = {
+  "java_opts": "-Djava.awt.headless=true -Djenkins.install.runSetupWizard=false",
+}
+volume_name = "jenkins-volume"
+register_consul_service = true
+plugins = ["configuration-as-code", "hashicorp-vault-plugin", "job-dsl", "nomad"]
+jasc_config = <<EOF
+jenkins:
+  agentProtocols:
+  - "JNLP4-connect"
+  - "Ping"
+  clouds:
+  - nomad:
+      name: "nomad"
+      nomadUrl: "http://{{ env "attr.unique.network.ip-address" }}:4646"
```

Comment:
here I would add the consul service variant, if the job is registering itself to consul.